business/jaeger_helper.go

```diff
@@ -77,6 +77,10 @@ func getErrorTracesFromJaeger(namespace string, service string, requestToken str
 	return errorTraces, err
 }
 
+func GetJaegerInternalURL(path string) (*url.URL, error) {
+	return url.Parse(fmt.Sprintf("http://%s%s%s", appstate.JaegerConfig.Service, appstate.JaegerConfig.Path, path))
```

Comment:
The URL is built from service without port ... is that supposed to work ?
For me it seems it doesn't. E.g. from kiali pod I have to curl on `http://jaeger-query:16686/...` and not just `http://jaeger-query/...`